### PR TITLE
859_closing_issues: update git vignette

### PR DESCRIPTION
**Pull request by bundfussr**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://roche-gsk.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://roche-gsk.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://roche-gsk.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::documents()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://roche-gsk.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

File: vignettes/git_usage.Rmd [hunk 1]
```diff
@@ -96,16 +99,25 @@ Once you have completed the Pull Request you will see all committed changes are
...
+**Delete branch** in the Pull Request.
 
 ```{r, echo = FALSE}
     knitr::include_graphics("github_delete_branch.png", dpi = 144)
 ```
 
+After merging the Pull Request the corresponding issue must be moved to the
+"Sprint Done" column on the "admiral scrum board". The issue must not be closed.
+It will be closed automatically once the `devel` branch is merged to the `main`
+branch.
```
Comment (bms63) [inline vignettes/git_usage.Rmd:120]: Do we want to specify who is supposed to do this?  The Reviewer, the person who made the PR, the scrum master?

File: vignettes/git_usage.Rmd [hunk 2]
```diff
@@ -88,6 +88,9 @@ Once all changes are committed, push the updated branch to GitHub:
   
 In GitHub, under **Pull requests**, the user will either have a "Compare and pull request" button and/or a "Create Pull Request".  The first button will be created for you if GitHub detects recent changes you have made. The branch to merge with must be the `devel` branch (base = `devel`) and the compare branch is the new branch to merge - as shown in the below picture. Please **pay close attention** to the branch you are merging into!  
 
+The issue must be linked to the pull request in the "Linked issues" field of the
+Pull Request.
```
Comment (bms63) [inline vignettes/git_usage.Rmd:92]: Should we have an image here?

File: vignettes/git_usage.Rmd (hunk 1 above)
Comment (bms63) [reply vignettes/git_usage.Rmd:120]: Also do we want an image or two here showing the Project board and moving stuff?  Umm...now I'm nervous about people accessing the board and messing things up??

Comment (bundfussr) [reply vignettes/git_usage.Rmd]: I have added the responsible person and a screenshot.

You can move the issue without going to the project board. This should avoid messing up the board.

File: vignettes/git_usage.Rmd (hunk 2 above)
Comment (bundfussr) [reply vignettes/git_usage.Rmd:92]: Yes, I have added one.

**Review by rossfarrugia — APPROVED** [review]

**Comment by rossfarrugia** [comment]
Looks good @bundfussr! I look forward to also seeing @bms63's PR checklist updates to this vignette for https://github.com/pharmaverse/admiral/issues/817 which will then complete the guide nicely.

**Review by thomas-neitmann — APPROVED** [review]

**Pull request merged**